paper-api/src/main/java/org/bukkit/inventory/view/builder/InventoryViewBuilder.java

```diff
@@ -0,0 +1,53 @@
+package org.bukkit.inventory.view.builder;
+
+import net.kyori.adventure.text.Component;
+import org.bukkit.entity.HumanEntity;
+import org.bukkit.inventory.InventoryView;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Generic Builder for InventoryView's with no special attributes or parameters
+ *
+ * @param <V> the type of InventoryView created from this builder
+ */
+@ApiStatus.Experimental
+public interface InventoryViewBuilder<V extends InventoryView> {
+
+    /**
+     * Makes a copy of this builder
+     *
+     * @return a copy of this builder
+     */
+    @NotNull
+    InventoryViewBuilder<V> copy();
+
+    /**
+     * Sets the title of the builder
+     *
+     * @param title the title
+     * @return this builder
+     * @deprecated use {@link #title(Component)} instead.
+     */
+    @Deprecated(forRemoval = true)
+    @NotNull
+    InventoryViewBuilder<V> title(@NotNull final String title);
```

Comment:
Why add deprecated methods?
You already have components for titles so no need for legacy methods anyway, or am I missing something?

Reply:
This is up for internal discussion, regarding spigot compatibility this PR is a 1 to 1 copy of an upstream feature I added thus I copied it one for one and instead added adventure components. Its very likely a core team discussion will dictate the future of these methods. I am not here to make these decisions.

Reply:
Yea, we won't need legacy string methods :+1: